app/templates/attendee-app.hbs

```diff
@@ -0,0 +1,89 @@
+  <a class='active item'>
+    {{t 'Attendee App'}}
+  </a>
+</div>
+<div class="ui segment">
+  <div class="content">
+    <img class="ui fluid image" src="/images/attendee-app.jpg" alt="Open Event Attendee App">
+  </div>
+  <h1 class="weight-500 ui huge center aligned header">
+    {{t 'Open Event Attendee App'}}
+  </h1>
+  <div class="ui grid">
+    <div class="five wide column">
+    </div>
+    <div class="three wide column">
+      <a class="store" href='https://play.google.com/store/apps/details?id=com.eventyay.attendee'><img alt='Get it on Google Play' src='/images/google-play.png' height="80"></a>
+    </div>
+    <div class="three wide column">
+      <a class="store" href='https://f-droid.org/en/packages/com.eventyay.attendee/'><img alt='Get it on F-Droid' src='/images/fdroid.png' height="80"></a>
+    </div>
+    <div class="five wide column">
+    </div>
+  </div>
+  <div class="ui grid">
+    <div class="eight wide column">   
+      <h1 class="weight-900">
+        {{t 'Get Your Next Experience of Open Event'}}
+      </h1>
+      <div class="ui hidden divider"></div>
+      <div class="content">
+        <h2 class="weight-600">
+          {{t 'Get More'}}
+        </h2>
+        <h3 class="weight-400 ui muted text">
+          {{t 'Check popular events on the go,'}} 
+          {{t 'unique , local , near you just for you'}}
+        </h3>
+      </div>
+      <div class="ui hidden divider"></div>
+      <div class="content">
+        <h2 class="weight-600"> {{t 'Know More'}}</h2>
+        <h3 class="weight-400 ui muted text">
+          {{t 'Check all event related information on the go and'}}
+          {{t 'get all relevant details about the speakers'}}
+        </h3>
+      </div>
+      <div class="ui hidden divider"></div>
+      <div class="content">
+        <h2 class="weight-600"> 
+          {{t 'Manage More'}}
+        </h2>
+        <h3 class="weight-400 ui muted text">
+          {{t 'Manage all you ticket details , event details '}} 
+          {{t 'on the go with your phone'}}
+        </h3>
+      </div>
+      <div class="ui hidden divider"></div>
+      <div class="content">
+        <h2 class="weight-600">
+          {{t 'Go Digital'}}
+        </h2>
+        <h3 class="weight-400 ui muted text">
+          {{t 'Show your ticket details , Session Pass on your phone using QR code'}} 
+          {{t 'and get in'}}
+        </h3>
+      </div>
+      <div class="ui hidden divider"></div>
+      <div class="content">
+        <h2 class="weight-600">
+          {{t 'Go Hassle-free'}}
+        </h2>
+        <h3 class="weight-400 ui muted text">
+          {{t 'Pay with your phone , Register for nearby events with your phone'}} 
+          {{t 'hasslefree , easily and conveniently'}}
+        </h3>
+      </div>
+    </div>
+    <div class="eight wide column">
+      <img class="ui large image" src="/images/android-img.jpg" alt="Android Pic">
+    </div>
+  </div>
+</div>
+<div class="ui center aligned muted text">
```

Comment:
add "header" to the class name in the end, to make it similar to organizer landing page

Reply:
@uds5501 Didnt notice the icons were collapsing in mobile view ,  👍

Reply:
@uds5501 Have a look now

Reply:
@kushthedude 
- Decrease the icon size on mobile view
![Screenshot from 2019-05-20 22-19-47](https://user-images.githubusercontent.com/21087061/58038521-c6d42180-7b4d-11e9-865a-254ae8a284e6.png)

- Improve alignment of these paragraphs? They look odd.
![Screenshot from 2019-05-20 22-20-14](https://user-images.githubusercontent.com/21087061/58038547-d81d2e00-7b4d-11e9-97b3-4ed2e85ed2b5.png)

Reply:
@uds5501 Check them now

Reply:
@kushthedude I asked the paragraphs to be aligned in the mobile view , not completely removed :sweat_smile: